nanopub/client.py

```diff
@@ -172,14 +171,15 @@ def fetch(uri, format: str = 'trig'):
         else:
             raise ValueError(f'Format not supported: {format}, choose from '
                              f'{[format.value for format in Formats]})')
-
+        if self.use_test_server:
```

Comment:
This has the implication that whenever a user creates a NanopubClient(use_test_server=True) you can only fetch test nanopublicaiton. I kind of like that, you are sort of in the test environment, but I'm curious what the reviewer thinks.